listenbrainz/mbid_mapping/mapping/mb_metadata_cache.py

```diff
@@ -61,6 +61,8 @@ def __init__(self, mb_conn, lb_conn=None, batch_size=None):
         super().__init__("mapping.mb_metadata_cache", mb_conn, lb_conn, batch_size)
 
     def get_create_table_columns(self):
+        # this table is created in local development and tables using admin/timescale/create_tables.sql
```

Comment:
I'm glad you got the note in both places!

Reply:
:)